src/components/TrendingTrack.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import styles from "./TrendingTrack.module.css";
+import playButton from "../images/play.svg";
+import likeButton from "../images/heart.svg";
+import moreButton from "../images/dots.svg";
+
+const TrendingTrack = ({ track }) => {
```

Comment:
I think we should rename this to just something like 'SongThumbnail', since it's likely we might reuse this outside of the trending section